alerts/project_triggers.go

```diff
@@ -178,6 +178,13 @@ func getShouldExecuteError(ctx triggerContext) message.Fields {
 			"variant": ctx.task.BuildVariant,
 			"project": ctx.task.Project,
 		},
+		"previous": map[string]interface{}{
+			"id":          ctx.previousCompleted.Id,
```

Comment:
This could be a nil pointer. Could accessing its fields (`Id`, `BuildVariant`, ...) cause a nil pointer exception?